backend/src/blockchain/TransactionToIota.ts

```diff
@@ -0,0 +1,161 @@
+  if (recipientGroupAlias !== '' && resultPackTransaction.data.transactions.length > 1) {
+    sodium.crypto_sign_detached(
+      sign,
+      Base64.toUint8Array(resultPackTransaction.data.transactions[1].bodyBytesBase64),
+      privateKey,
+    )
+    const resultSendTransactionIota = await apiPost(
+      CONFIG.BLOCKCHAIN_CONNECTOR_API_URL + 'sendTransactionIota',
+      {
+        bodyBytesBase64: resultPackTransaction.data.transactions[1].bodyBytesBase64,
+        signaturePairs: [
+          {
+            pubkey: senderPubkeyHex,
+            signature: sign.toString('hex'),
+          },
+        ],
+        groupAlias: recipientGroupAlias,
+      },
+    )
+    // throw error if something went wrong
+    if (!resultSendTransactionIota.success) {
+      throw new Error(resultSendTransactionIota.data)
+    }
+  }
+  return signature
+}
+
+async function sendCoins(
+  created: Date,
+  user: User,
+  apolloTransactionId: number,
+  recipientPublicHex: string,
+  amount: string,
+  memo: string,
+  recipientGroupAlias: string | '',
+): Promise<Buffer> {
+  const privateKey = await recoverPrivateKey(user)
+  const encryptedMemo = encryptMemo(memo, privateKey, Buffer.from(recipientPublicHex, 'hex'))
+  const packTransactionRequest = {
+    transactionType: 'transfer',
+    created: created.toISOString(),
+    apolloTransactionId: apolloTransactionId,
+    senderPubkey: user.pubKey.toString('hex'),
+    recipientPubkey: recipientPublicHex,
+    amount: amount,
+    memo: encryptedMemo,
+    senderGroupAlias: '',
+    recipientGroupAlias: '',
+  }
+
+  return signAndSendTransaction(user, privateKey, packTransactionRequest, recipientGroupAlias)
+}
+
+async function registerNewGroup(
+  created: Date,
+  user: User,
+  communityName: string,
+  communityAlias: string,
+  communityCoinColor: number | string,
+): Promise<Buffer> {
+  const privateKey = await recoverPrivateKey(user)
+  const packTransactionRequest = {
+    transactionType: 'groupAdd',
+    created: created.toISOString(),
+    groupName: communityName,
+    groupAlias: communityAlias,
+    coinColor: communityCoinColor,
+  }
+  return signAndSendTransaction(user, privateKey, packTransactionRequest, '')
+}
+
+async function creation(
+  created: Date,
+  signingUser: User,
+  recipientUser: User,
+  apolloTransactionId: number,
+  memo: string,
+  amount: string,
+  targetDate: Date,
+): Promise<Buffer> {
+  const privateKey = await recoverPrivateKey(signingUser)
+  const encryptedMemo = encryptMemo(memo, privateKey, recipientUser.pubKey)
+  const packTransactionRequest = {
+    transactionType: 'creation',
+    created: created.toISOString(),
+    apolloTransactionId: apolloTransactionId,
+    memo: memo,
+    recipientPubkey: recipientUser.pubKey.toString('hex'),
+    amount: amount,
+    targetDate: targetDate.toISOString(),
+  }
+  return signAndSendTransaction(signingUser, privateKey, packTransactionRequest, encryptedMemo)
+}
+
+export { sendCoins, registerNewGroup, creation }
```

Comment:
Do I understand it right:
This are the connector methods that need to be added to the TransactionResolver for sendCoins and AdminResolver for creation?
A bit like registerNewGroup thas has been added to ctrateServer!?

Reply:
Yes